### PR TITLE
Rubocop improve

### PR DESCRIPTION
**Pull request by pocke**
I want to use SideCI about Rubocop.
But SkyHopper has many warnings for rubocop.

I fixed the warnings.
And Add some rules.

Rules
------

- check indent
- should not use hash rocket :rocket: 

If this PR is merged, I'll enable comment to GitHub by SideCI about Rubocop.

### CODE REVIEW

**Comment by Joeper214** [comment]
:clap:

**Comment by pocke** [comment]
@Joeper214  Please review and merge this!

**Comment by Joeper214** [comment]
@pocke . I'll review it tomorrow.

File: app/controllers/dishes_controller/validate.rb [hunk 1]
```diff
@@ -10,12 +10,11 @@ module DishesController::Validate
   # POST /dishes/1/validate
   def validate
     id = params.require(:id)
-    dish  = Dish.find(id)
```
Comment (Joeper214) [inline app/controllers/dishes_controller/validate.rb]: Is this unused dish?

**Comment by Joeper214** [comment]
LGTM!

**Comment by Joeper214** [comment]
Merged Thanks!

**Comment by pocke** [comment]
Thanks :+1:

**Pull request merged**